### DISCUSSION TITLE
Triaging issues and PRs

### DISCUSSION THREAD

**Discussion by mblarsen** [livewire/livewire#1333] [4. General]
Hey all,

I noticed that quite a lot of issues and PRs are piling up. Is there any form of triage process in place. E.g. assigning bug, feature, working-on, and so on for issues and PRs? I think this could prevent issues from getting lost and it will possibly reduce the time the owners/maintainers have to spend on reviewing.

What are your thoughts?

**Comment by lancepioch** [comment]
Issues are currently for reproducible bugs.
Pull requests are for code changes, new features, etc.
Discussions are for everything else.

**Reply by mblarsen** [reply]
That's my point. That is a good set of intentions/guidelines, but it isn't like that if you scan through the issues. In issues you'll find: bugs, how-do-I-s, feature requests, and what not. Not all issues are equal. Triaging is about making this clear. One approach would be just to label them with 'bug', 'feature'. One could go further by assigning labels for which version the fix or feature belongs in, e.g. 1.x, 2.x.

In addition to labelling one could covert (or ask to convert) issues to discussions where appropriate. That could be for questions of the howto character or discussions like this.

I 'unanswered' our 'answer' as this is a discussion and not a question for which there is one answer.

**Comment by mgussekloo** [comment]
Good point! Is there anything we can do to help the moderators and maintainers out? Less PR's getting piled up is beneficial to our entire community.
